app/steps/ui/declaration/template.html

```diff
@@ -10,60 +10,65 @@ <h1 class="form-title heading-large">{{ content.title }}</h1>
-        <p>{{ fields.legalStatement.value.deceasedEstateValue | safe }}</p>
-        <p>{{ fields.legalStatement.value.deceasedEstateLand | safe }}</p>
+        <h3 class="heading-small declaration-subheader">{{ content.deceasedEstateHeader }}</h3>
+        <p class="declaration-item">{{ fields.legalStatement.value.deceasedEstateValue | safe }}</p>
+        <p class="declaration-item">{{ fields.legalStatement.value.deceasedEstateLand | safe }}</p>
 
-        <h3 class="heading-small">{{ content.executorApplyingHeader }}</h3>
+        <h3 class="heading-small declaration-subheader">{{ content.executorApplyingHeader }}</h3>
         {% for executor in fields.legalStatement.value.executorsApplying %}
-            <p>{{ executor.name | safe }}</p>
-            <p>{{ executor.sign | safe }}</p>
+            <p class="declaration-item">{{ executor.name | safe }}</p>
+            <p class="declaration-item">{{ executor.sign | safe }}</p>
         {% endfor %}
 
         {% if fields.legalStatement.value.executorsNotApplying.length > 0 %}
-            <h3 class="heading-small">{{ content.executorNotApplyingHeader }}</h3>
+            <h3 class="heading-small declaration-subheader">{{ content.executorNotApplyingHeader }}</h3>
             {% for executor in fields.legalStatement.value.executorsNotApplying %}
-                <p>{{ executor | safe }}</p>
+                <p class="declaration-item">{{ executor | safe }}</p>
             {% endfor %}
         {% endif %}
 
-        <h2 class="heading-large">{{ content.declarationHeader }}</h2>
-        <p>{{ fields.declaration.value.confirm | safe }}</p>
+        <h2 class="heading-large declaration-subheader">{{ content.declarationHeader }}</h2>
+        <p class="declaration-item">{{ fields.declaration.value.confirm | safe }}</p>
 
         {{ unorderedList([
             fields.declaration.value.confirmItem1,
             fields.declaration.value.confirmItem2,
             fields.declaration.value.confirmItem3
         ]) }}
 
-        <p>{{ fields.declaration.value.requests | safe }}</p>
+        <p class="declaration-item">{{ fields.declaration.value.requests | safe }}</p>
 
         {{ unorderedList([
             fields.declaration.value.requestsItem1,
             fields.declaration.value.requestsItem2
         ]) }}
 
-        <p>{{ fields.declaration.value.understand | safe }}</p>
+        <p class="declaration-item">{{ fields.declaration.value.understand | safe }}</p>
 
         {{ unorderedList([
             fields.declaration.value.understandItem1,
             fields.declaration.value.understandItem2
         ]) }}
 
     </div>
-
+    <div class="declaration-grid-item">
+        <h2 class="heading-medium">{{ content.saveThisPage }}</h2>
+        <p><img class="icon-file-download" alt="Download PDF document"/>&nbsp;
+            <a id="declarationPdfHref" href="/declarationPdf">{{content.declarationPdf}}</a></p>
+    </div>
```

Comment:
Shouldn't we be using a feature toggle for this link?

Reply:
It was discussed whether to use a feature but it was decided that we would deploy the corresponding business service pr prior too the front end